jfrog_metrics/datadog_checks/jfrog_metrics/check.py

```diff
@@ -0,0 +1,47 @@
+from datadog_checks.base import ConfigurationError, OpenMetricsBaseCheck
+
+
+class JfrogMetricsCheck(OpenMetricsBaseCheck):
+    """
+    Collect metrics from JFrog
+    """
+
+    def __init__(self, name, init_config, instancetype, instances=None):
```

Comment:
```suggestion
    def __init__(self, name, init_config, instances=None):
```
Let's have `instancetype` as a configuration option in the `spec.yaml` file

Reply:
Sure i have committed necessary changes in spec and tests for instanceType